kahuna/public/js/components/gr-collections-panel/gr-collections-panel-node.html

```diff
@@ -19,31 +19,48 @@
         </button>
 
         <a class="node__name flex-spacer"
-           ui:sref="search.results({query: (node.data.name)})">
+           ui:sref="search.results({query: ctrl.getCollectionQuery(node.data.content.pathId)})">
             {{node.data.name}}
         </a>
 
+
         <button class="node__action clickable" type="button"
                 ng:if="ctrl.deletable"
-                ng:click="ctrl.remove()">
+                ng:show="ctrl.editing"
```

Comment:
Not sure about ngDisabled either, since the editing mode is transforming the UI rather than enabling functionality that we otherwise want present but disabled.

Re if vs show, I don't think it's a big deal, but given how infrequently people will toggle to editing mode, it doesn't necessarily feel worth rendering extra DOM elements (and directives) for every node visible in the possibly large tree at all times. Up to you though.